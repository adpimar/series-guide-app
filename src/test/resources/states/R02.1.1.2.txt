S#321060#The OA#Continuing#2016-12-16#Friday#12:00 AM#Netflix#Drama, Mystery, Science-Fiction#8.1#18#Having gone missing seven years ago, the previously blind Prairie Johnson returns home, now in her 20s with her sight restored. While many believe she is a miracle, others worry that she could be dangerous.
S#260449#Vikings#Continuing#2013-03-03#Wednesdey#09:00 PM#History#Action, Drama#8.0#441#Vikings follows the adventures of Ragnar Lothbrok the greatest hero of his age. The series tells the sagas of Ragnar's band of Viking brothers and his family, as he rises to become King of the Viking tribes. As well as being a fearless warrior, Ragnar embodies the Norse traditions of devotion to the gods, legend has it that he was a direct descendant of Odin, the god of war and warriors.
S#262368#Vikings (2012)#Ended#2012-09-11#Tuesday#09:00 PM#BBC Two#Documentary#6.0#13#History sees the Vikings as a band of bloodthirsty pirates, raiding peaceful Christian monasteries... and it's true. The vikings took no prisoners, relished cruel retribution and prided themselves as fierce warriors. But their Prowess in battle is only the start of the story. Going on the trail of the real Vikings this series reveals an extraordinary story of a people who, from the brink of destruction, built an empire reaching around a qarter of the globe. Where did they come from? How did they really live? And what drove them to embark on such extraordinary voyages of discovery? Neil Oliver goes beyond their bloody reputation, from Iceland to Instanbul, to search for the truth about the Vikings.