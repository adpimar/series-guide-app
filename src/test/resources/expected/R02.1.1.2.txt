S#260449#Vikings#Continuing#2013-03-03#Wednesdey#09:00 PM#History#Action, Drama#8.0#441#Vikings follows the adventures of Ragnar Lothbrok the greatest hero of his age. The series tells the sagas of Ragnar's band of Viking brothers and his family, as he rises to become King of the Viking tribes. As well as being a fearless warrior, Ragnar embodies the Norse traditions of devotion to the gods, legend has it that he was a direct descendant of Odin, the god of war and warriors.